test/runtests.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# runtests.sh
```

Comment:
@ndt93 Sorry for the extra space, fixed it and changed the permission

> On 26 Aug 2016, at 3:34 PM, Thien Nguyen <notifications@github.com> wrote:
> 
> In test/runtests.sh <https://github.com/se-edu/addressbook-level2/pull/29#discussion_r76375887>:
> 
> > @@ -0,0 +1,29 @@
> > +#!/bin/bash
> > +# runtests.sh
> No need to put the file name here? :-P
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/se-edu/addressbook-level2/pull/29/files/b0aabaad0d0545207eb0fb7a43b7b9f375608a37#r76375887>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AL1PPPRKuAjua-RmgSNaSe-JO98Z0C9Xks5qjpcMgaJpZM4JtqjD>.
>